Dockerfile.antigravity

```
@@ -0,0 +1,45 @@
+# Antigravity Python Service
+# Provides multi-agent swarm orchestration for RTGS Sales Automation
+
+FROM --platform=linux/amd64 python:3.11-slim
+
+# Install Node.js 20 (required for MCP stdio transport)
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)